src/DynamoCoreWpf/Controls/ShortcutToolbar.xaml

```diff
@@ -139,6 +140,28 @@
                 <Menu HorizontalAlignment="Right"
                       Style="{StaticResource MainMenu}"
                       IsMainMenu="True">
+                    <MenuItem Name="MenuItem">
+                        <MenuItem.Header>
+                                <Button x:Name="notificationsButton" 
+                                        BorderBrush="Transparent"                                         
+                                        Background="Transparent">
+                                    <DockPanel>
+                                        <fa:FontAwesome Margin="0,0,20,0"  DockPanel.Dock="Bottom" Icon="BellOutline" Foreground="#38ABDF" FontSize="20px"/>
+                                        <Border DockPanel.Dock="Top" Background="#38ABDF" CornerRadius="2" 
```

Comment:
I can see that 38ABDF is already defined in DynamoModern.xaml an DynamoColorsAndBrushes.xaml so you can reuse it.